buildscripts/kokoro/windows.bat

```diff
@@ -31,12 +33,16 @@ echo failOnWarnings=true>> gradle.properties
 echo vcProtobufLibs=%ESCWORKSPACE%\\grpc-java-helper\\protobuf-%PROTOBUF_VER%\\cmake\\build\\Release>> gradle.properties
 echo vcProtobufInclude=%ESCWORKSPACE%\\grpc-java-helper\\protobuf-%PROTOBUF_VER%\\cmake\\build\\include>> gradle.properties
 
+@echo on
```

Comment:
Instead of echo on and echo off, just prefix the command with `@`. Like `@cmd.exe ...`